components/_patterns/02-molecules/date-bars/date-bar.twig

```diff
@@ -8,12 +8,16 @@
 
 {% set date_bar_base_class = 'date-bar' %}
 
-<ul {{ bem(date_bar_base_class , (date_bar_modifiers), date_bar_blockname) }}>
+{% if date(event_date_end) < date() %}
+  {% set date_bar_modifiers = ['ended'] %}
+{% endif %}
+
+<div {{ bem(date_bar_base_class , (date_bar_modifiers), date_bar_blockname) }}>
   {% if date(event_date_end) < date() %}
-    <li>The event has ended</li>
+    The event has ended
   {% else %}
-    <li>{{ event_date_start|date("l, F d") }}</li> 
+    <span>{{ event_date_start|date("l, F d") }}</span> 
```

Comment:
I really try not to style elements (or nested selectors) at all costs. Sometimes it's unavoidable, but in this case, I would add classes to the spans (even if it's just the same class).

Something like:
`<span {{ bem('date', [], date_bar_blockname) }}>`

then you can replace line 15 in the stylesheet with
`  &__date {`

Reply:
@ModulesUnraveled I think we're actually going to use a header here. Two questions:

1. Is it possible to have an `h2` with an `h3` class applied? (Semantically we want it to have a certain level but visually look differently.)
2. How do you pass modifiers to the headers component?

Reply:
Yeah, you can style an h2 to look like whatever you want. Including an h3. I'd recommend making the h3 styles a mixin, and then you can just apply that mixin to the class you set on the h2.

Here's an example of how to include a heading. You'll change the `heading_level` to 2, and then set the class to be whatever you want.

https://github.com/NYPL/nypl_emulsify/pull/7/files#diff-8749a035dbe51bcb4492fc05c7fe4c2cR6

Reply:
Done, gonna resolve this.